src/main/resources/hudson/plugins/throttleconcurrents/ThrottleStep/config.jelly

```diff
@@ -0,0 +1,7 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define"
+         xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:entry title="${%Category}" field="category">
+        <f:textbox/>
+    </f:entry>
```

Comment:
BTW, I am perfectly fine with dropping the support of local job throttling without categories